redpen-core/src/main/java/cc/redpen/validator/JavaScriptValidator.java

```diff
@@ -224,6 +224,7 @@ protected String getLocalizedErrorMessage(String key, Object... args) {
                         "var addLocalizedError = Function.prototype.bind.call(redpenToBeBound.addLocalizedError, redpenToBeBound);" +
                         "var addLocalizedErrorFromToken = Function.prototype.bind.call(redpenToBeBound.addLocalizedErrorFromToken, redpenToBeBound);" +
                         "var addLocalizedErrorWithPosition = Function.prototype.bind.call(redpenToBeBound.addLocalizedErrorWithPosition, redpenToBeBound);");
```

Comment:
I'd love to do so, but that will break a test.  

In redpen-core/src/test/java/cc/redpen/validator/JavaScriptValidatorTest.java, we have:

    @Test
    public void testJSLiteralValidator() throws RedPenException, IOException {
        ...
        validator.scripts.add(new UnconfinedScript(validator, "testScript.js",
                "function preValidateSentence(sentence) {" +
                        // add function names to "calledFunctions" list upon function calls for the later assertions
                        // the following script is using Nashorn's lobal object "Java".type to access static member:
                        // http://docs.oracle.com/javase/8/docs/technotes/guides/scripting/nashorn/api.html
                        "Java.type('cc.redpen.validator.JavaScriptValidatorTest').calledFunctions.add('preValidateSentence');}" +
         ...

So the test function needs the bridging facility for legitimate purpose.

That's why I introduced UnconfinedScript.